tests/service/ProjectAnalysisTests.fs

```diff
@@ -2591,7 +2592,7 @@ let ``Test Project16 all symbols`` () =
             ("val x", "x", "file1", ((5, 11), (5, 12)), ["defn"], []);
             ("val x", "x", "file1", ((8, 11), (8, 12)), ["defn"], []);
             ("val x", "x", "file1", ((11, 11), (11, 12)), ["defn"], []);
-            ("Impl", "Impl", "file1", ((2, 7), (2, 11)), ["defn"], ["module"])|]
+            ("Impl", "Impl", "file1", ((2, 7), (2, 11)), ["defn"], ["module"])|] |> Array.sort)
```

Comment:
Could we maybe group/sort the usages in either the API or at least in the test helpers, so we don't have to do this in each test?

Reply:
Yeah that would make sense.

Reply:
Although there aren't really any test helpers here... 

And I don't know if we want to spend resources on sorting the results in the API when the ordering doesn't really matter. 

It would be nice to return a set instead. But that would mean an API change which probably isn't justified just for that. Maybe we could get that in if we need a new API for some more important reason.

Reply:
Well actually it was just reversed, not non-deterministic, so we can just keep it as it was :)